src/components/ChatLog.js

```diff
@@ -0,0 +1,41 @@
+import ChatEntry from './ChatEntry';
+import PropTypes from 'prop-types';
+import React from 'react';
+
+const ChatLog = (props) =>{
+    const chatMessagesData = props.entries.map(message => {  
+        return (
+        <div key={message.id}>
+            <ChatEntry
+            id={message.id}
+            sender={message.sender}
+            body={message.body}
+            timeStamp={message.timeStamp}
+            liked={message.liked}
+            onUpdateChat={props.onUpdateChat}
+            />
+        </div>
+        )
+    });
+    return (<div>
+        <div>{chatMessagesData}</div>
+        </div>
+        );
+};
+
+
+ChatLog.propTypes = {
+    entries: PropTypes.arrayOf(PropTypes.shape({
+        id: PropTypes.number.isRequired,
+        sender: PropTypes.string.isRequired,
+        body: PropTypes.string.isRequired,
+        timeStamp:PropTypes.string.isRequired,
+        liked: PropTypes.bool.isRequired
+
+    })),
+    onUpdateChat: PropTypes.func.isRequired
+
+};
```

Comment:
Yesssss! This is literally so perfect! Great job you all!!! You all did amazing work! Especially on setting up the lifting up state! Understanding how to utilize it is easier than the set up so be proud of yourselves!